coralproject/talk
# [CORL-938] Ensure comments are still visible when stream is closed

## What does this PR do?

Removes an unnecessary check for showing the comment stream only if the comment is open.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to stream as an admin
- Go to Config and Close the stream
- Go back to the comment streams/tabs
- See that you can view the comments but not reply/comment
